.github/workflows/cd.yaml

```diff
@@ -0,0 +1,43 @@
+name: CD
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.15.1]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2.1.4
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: ./script/build
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.15.1]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2.1.4
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: ./script/test
+  release:
```

Comment:
I know this doesn't matter until we _actually_ release automatically, but by default these [jobs will run in parallel](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs), which probably isn't what we want here. 

We could either refactor so these all run as steps in a single job (which will run sequentially), or add the [`needs` syntax](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds) to the release job.

Happy for that to happen when we add the automated release step, as long as you think we'll remember!

Reply:
Didn't know about `needs`, nice! I think it would be nice to have a single `job` which simply runs `./script/cd`??? However that's different from the approach taken in the CI step, so for consistency have added `needs`.